projects/plugins/jetpack/modules/sso.php

```diff
@@ -119,7 +362,7 @@ public function sso_reminder_logout_wpcom( $errors ) {
 		if ( ! empty( $errors->errors['loggedout'] ) ) {
 			$logout_message = wp_kses(
 				sprintf(
-					/* translators: %1$s is a link to the WordPress.com account settings page. */
```

Comment:
Would you mind reverting these whitespace changes? They happen to me all the time, too, where my IDE has a different idea about how these things should be aligned on auto-formatting and WPCS can't change that.

Reply:
Reverted the whitespace changes here.

Reply:
Sorry, I meant all whitespace changes in that file

Reply:
I think this will be an uphill battle to maintain these spaces, the standards now seem to demand these spaces now and even running `vendor/bin/phpcbf projects/plugins/jetpack/modules/sso.php` yields the same results. Am I missing something?